drafts/draft-0005.md

```diff
@@ -0,0 +1,188 @@
+
+Response fields:
+
+Name | Type | Description
+-----|------|------------
+`id` | string | Required. An identifier for the multisig request that is generated by the multisig service.
+`status` | string | Required. An string indicating the status of the multisig request which can have one of the following values:<br><br>- `pending` - The request requires more signatures<br>- `success` - The transaction has been fully signed<br>- `failed` - There was an error with the transaction and it could not be submitted successfully
+`uri` | string | Required. The SEP-0007 URI that was submitted to the service
+`signers`| string[] | Required. An array of public keys that have signed the transaction. 
+
+
+Example:
+
+```json
+{
+  "id": "c585ceee-b1b9-4009-aa37-b8544346a036",
+  "status": "pending",
+  "uri": "web+stellar:tx?xdr=AAAAAHFd0%2BHQV5u%2FY%2FfM3%2BVelUr1IWwSFL8CUDIAUudUdD4MAAAAZAADyI8AAAADAAAAAAAAAAAAAAABAAAAAAAAAAEAAAAAcV3T4dBXm79j98zf5V6VSvUhbBIUvwJQMgBS51R0PgwAAAAAAAAAAACYloAAAAAAAAAAAVR0PgwAAABAEw8ODG0iixkbAHg1aJATAnZS2531PhGauuSvFDad2WxHKzIenUNbc7K5mGiSpe5jvqe19OQCbNFuBjqN11jfBw%3D%3D",
+  "signers": ["GBYV3U7B2BLZXP3D67GN7ZK6SVFPKILMCIKL6ASQGIAFFZ2UOQ7AZB5V"]
+}
+```
+
+####  CORS headers
+
+In order to comply with browser cross-origin access policies, the service should provide wildcard CORS response HTTP header. The following HTTP header must be set for the API endpoints:
+
+```
+Access-Control-Allow-Origin: *
+```
+
+### Multisig Transaction Flow
+
+1. Determine whether a transaction requires additional signatures in order to be valid.
+1. If the transaction requires more signatures, look up the data entry with the key `multisig.domain` on the transaction source account.
+1. Resolve the `stellar.toml` associated with the account's `multisig.domain` entry and return the `MULTISIG_ENDPOINT` field.
+1. Convert the transaction to a SEP-0007 style URI, setting any additional parameters such as `callback`.
+1. Before submitting the transaction, it should contain at least one valid signature.
+1. `POST` to the `MULTISIG_ENDPOINT` endpoint with the SEP-0007 `tx` URI in the body as the parameter `uri`.
+1. The service responds with a 200 HTTP status code and a JSON object specified in the Submit Transaction Endpoint Response section.
+
+## Rationale
+
+When implementing StellarGuard for the Stellar Account Viewer, it was rightly pointed out it should avoid vendor-specific implementation. This SEP provides an alternative approach that can be used by tools that want to support multisig accounts without implementing a multisig server for each product. 
+
+One design choice was whether to skip the `stellar.toml` step and add the fully qualified multisig server endpoint to the account's data field. The `stellar.toml` approach affords greater flexibilty for the server to change its implementations. It is much easier to update the `stellar.toml` than to update the data entry of every account that uses it.
+
```

Comment:
I'm still wondering about that one: URL changing from things such as coordinator are not a very welcome event.

Reply:
Ok don't want to overthink on that one.